### PR TITLE
#2121 - Add hindi localization support for a few patterns

### PR DESCRIPTION
**Pull request by rishu1601**
Added the hindi localization for:
- adpater
- factory
- observer
- prototype
- singleton
- strategy

### CODE REVIEW

**Comment by rishu1601** [comment]
Hey @iluwatar , waiting for your review

**Comment by rishu1601** [comment]
Hi @iluwatar , please review this PR

**Comment by rishu1601** [comment]
Hi @iluwatar

**Comment by iluwatar** [comment]
Part of issue #2279

File: localization/hi/adapter/README.md [hunk 1]
```diff
@@ -0,0 +1,141 @@
+---
+layout: pattern
+title: Adapter
+folder: adapter
+permalink: /patterns/adapter/hi
+categories: Structural
+language: hi
+tags:
+- Gang of Four
+---
```
Comment (iluwatar) [inline localization/hi/adapter/README.md:10]: Have a look at the updated yaml frontmatter requirements at https://github.com/iluwatar/java-design-patterns/wiki/01.-How-to-contribute. You can drop some of the tags. Check this for all the patterns in this PR.

**Review by iluwatar — CHANGES_REQUESTED** [review]
Additionally, there is a conflict preventing the merge. Please resolve it.

**Pull request closed without merging**